src/esf-mail/esf-mail-notification.ts

```diff
@@ -18,9 +18,11 @@ export function setupEsfNotificationMail(
   domainName: string,
 ) {
 
-  // Backup message to S3 bucket
-  const backupBucket = new s3.Bucket(scope, 'esf-mail-notification-s3-backup', {
-    bucketName: statics.s3Name_esfMailNotificationBackupBucketName,
+  /**
+     * Bucket to backup all SES messages
+     */
+  const backupBucket = new s3.Bucket(scope, 'ses-mail-s3-backup', {
+    bucketName: statics.s3Name_sesMailBackupBucketName,
     removalPolicy: RemovalPolicy.DESTROY,
```

Comment:
Nu prima, is dit in prod een issue?

Reply:
Hebben we ze na 180 dagen nog nodig in prod?